test/verify/check-metrics

```diff
@@ -416,8 +416,8 @@ class TestHistoryMetrics(testlib.MachineCase):
         # Now add the journal
         # Journal was recorded on Fedora 33 and when trying to use it with older systemd it fails with:
         # `Journal file /var/log/journal/*/journal.journal uses an unsupported feature, ignoring file.`
-
-        if self.machine.image == "centos-8-stream" or self.machine.image.startswith("rhel-8-"):
+        # Also, older systemds get a slightly wrong log window with --since/until
+        if re.search(r"centos-[89]|rhel-[89]|debian-stable|ubuntu-2204|ubuntu-stable", self.machine.image):
```

Comment:
I almost wonder why we don't do version detection of systemd?

Reply:
Two reasons from my POV: systemd has fractional versions and lots of distro patches, and I don't want to carry the old test variant around for a couple of years.